src/main/java/com/sivalabs/springblog/comment/domain/CommentEntity.java

```diff
@@ -0,0 +1,105 @@
+package com.sivalabs.springblog.comment.domain;
+
+import com.sivalabs.springblog.post.domain.PostEntity;
+import com.sivalabs.springblog.user.domain.UserEntity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "comments")
+public class CommentEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "comment_id_seq_gen")
+    @SequenceGenerator(name = "comment_id_seq_gen", sequenceName = "comment_id_seq", allocationSize = 50)
```

Comment:
As you are using default allocation size, line 23 and 23 can be simply replaced with  `@GeneratedValue(strategy = GenerationType.SEQUENCE)` if you are fine with sequenceName generated OOTB as `comments_seq`

Reply:
Thanks for the suggestion! I'm explicitly using `@SequenceGenerator` because I want to ensure the sequence name (`comment_id_seq`) matches my database schema and use a specific allocationSize of 50.